src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -158,6 +163,32 @@ class Method5Builder[A, B, C, D, E, R](fb: FunctionBuilder[_], mname: String)
   def apply(a:Code[A], b: Code[B], c: Code[C], d: Code[D], e: Code[E]): Code[R] = invoke(a, b, c, d, e).asInstanceOf[Code[R]]
 }
 
+class ClassFieldRef[T: TypeInfo](fb: FunctionBuilder[_], name: String) extends Settable[T] {
```

Comment:
Should this be merged with `FieldRef` from `Code.scala`? (Even if we keep it separate, I think this class belongs in `Code.scala`)